# Generated for NaoQi method ALModule.exit
# Exits and unregisters the module.

---


